### PR TITLE
Order and subscription as part of customer create.

### PR DESCRIPTION
**Pull request by marcushanses**

### CODE REVIEW

File: docs/merchant/customer/create.md [hunk 1]
```diff
@@ -84,4 +84,87 @@ Authentication: Bearer <public.api.key | private.api.key>
...
 }
+```
+
+## With initial Orders and Subscriptions
+
+To add one or more [`Orders`](../order/reference.html#order) associated with the Customer, populate the `order` field with a list or a single Omit<[`Order.Creatable`](../order/reference.html#creatable), `"payment"` | `"customer"`>. The `items` value of the `order`, or the first `order` in the list, is used to authorize the [`Customer Method`](./reference.html#customermethod). 
+
+[Subscriptions](./reference.html#subscription) can be added in a similar way, by populating the Customer Creatable with [`Subscription Creatable`](./reference.html#subscription) or [`Subscription Creatable[]`](./reference.html#subscription).
+
+The response will be a [`Customer`](./reference.html#customer-2) & { order: `tokenized.order` | `tokenized.order`[] } and can be validated
```
Comment (Magnus93) [inline docs/merchant/customer/create.md:95]:
```suggestion
The response will be a [`Customer`](./reference.html#customer-2) & { order: `tokenized.order` | `tokenized.order[]` } and can be validated 
```
More consitent with line 93 (`Subscription Creatable[]`)

**Review by Magnus93 — CHANGES_REQUESTED** [review]

File: docs/merchant/customer/create.md [hunk 2]
```diff
@@ -84,4 +84,87 @@ Authentication: Bearer <public.api.key | private.api.key>
   "schedule": "monthly",
   "currency": "SEK"
 }
+```
+
+## With initial Orders and Subscriptions
+
+To add one or more [`Orders`](../order/reference.html#order) associated with the Customer, populate the `order` field with a list or a single Omit<[`Order.Creatable`](../order/reference.html#creatable), `"payment"` | `"customer"`>. The `items` value of the `order`, or the first `order` in the list, is used to authorize the [`Customer Method`](./reference.html#customermethod).
```
Comment (nilssonemma) [inline docs/merchant/customer/create.md:91]: I think we should avoid writing "Omit" since this is specific for Typescript

**Review by nilssonemma — CHANGES_REQUESTED** [review]

File: docs/merchant/customer/reference.md [hunk 3]
```diff
@@ -8,16 +8,17 @@ In the response from the endpoint, an object of the Customer data type will be r
...
+| `contact`      | [`Contact`](../common/reference.html#contact)                                        |                                                                             | Yes      |
+| `method`       | `Method.Creatable[]`                                                                 | See [`Method`](#customermethod) data type, may include an empty array       |          |
+| `currency`     | [`Currency`](../common/reference.html#currency)                                      | 3-letter currency code, e.g. "SEK"                                          | Yes      |
+| `schedule`     | [`Schedule`](./reference.html#schedule) or [`Frequency`](./reference.html#frequency) | see [`Schedule`](./reference.html#schedule) data type                       | Yes      |
+| `limit`        | `number`                                                                             | Credit limit in the specified currency for the customer                     | Yes      |
+| `order`        | [`Order | Order[]`](../order/reference.html#creatable)                               | Order or Orders to be created simultaneously as the customer                | Yes      |
```
Comment (nilssonemma) [inline docs/merchant/customer/reference.md:20]: Without the payment and customer field

File: docs/merchant/customer/create.md [hunk 4]
```diff
@@ -84,4 +84,89 @@ Authentication: Bearer <public.api.key | private.api.key>
...
+        }
+      }
+    }
+  ],
+  "contact": {
+    "name": "Joe Smith",
+    "email": "joe.smith@example.com"
+  },
+  "order": {
+	"items": 42,
```
Comment (Magnus93) [inline docs/merchant/customer/create.md:134]: Fix indentation

**Review by Magnus93 — CHANGES_REQUESTED** [review]

**Review by nilssonemma — APPROVED** [review]

**Review by fredrikbryntesson — APPROVED** [review]

**Pull request merged**